### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by george827**

### CODE REVIEW

**Comment by george827** [comment]
review

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+
+## Getting Started
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
```
Comment (franclobo) [inline README.md]: - [x] Kindly delete the sections that you are not using yet for a professional README file.

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+
+
+
+
+## Authors
+
+👤 **Author1**
```
Comment (franclobo) [inline README.md]: - [x] Please 🙏🏽 add your name to your project for a professional README file.

File: README.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+
+- GitHub: [@githubhandle]([https://github.com/githubhandle](https://github.com/george827))
+
+
+## Acknowledgments
+- Inspiration
```
Comment (franclobo) [inline README.md:22]: - [ ] Kindly add "Microverse" to the acknowledgments. 👍🏽

File: README.md [hunk 4]
```diff
@@ -1 +1,41 @@
...
+
+
+## 📝 License
+
+
```
Comment (franclobo) [inline README.md]: - [x] Please 🙏🏽 add the LICENSE file to your repo and make sure that the link works as expected for a professional README file.

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (franclobo) [inline .gitignore]: - [x] Kindly add the `test.md` file and the `node_modules/` folder to the `.gitignore` file.

File: README.md [hunk 6]
```diff
@@ -1 +1,41 @@
...
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle]([https://github.com/githubhandle](https://github.com/george827))
```
Comment (franclobo) [inline README.md]: - [x] Kindly review that the link works as expected for a professional README file.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @george827, 

Good job so far!
You are doing it good 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Descriptive commits ✔️ 
- Correct Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please 🙏🏽 add a title and description to your pull request.

> Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

image

image

- [x] Kindly review that all the linters pass without errors.
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**